.github/workflows/ci.yml

```diff
@@ -189,10 +193,22 @@ jobs:
           # setting up maven often times out on macOS
           maven: ${{ matrix.os != 'macos-latest' }}
       - uses: ./.github/actions/build-zeebe
+        id: build-zeebe
         with:
           go: false
-          maven-extra-args: -am -pl qa/integration-tests -T1C
-      - name: Run smoke test
+          maven-extra-args: -T1C
+      - uses: ./.github/actions/build-docker
+        id: build-docker
+        # container smoke tests are only run on linux which is the docker target platform
```

Comment:
:x: More interesting would be to document why :+1: I think just a short sentence would be enough. This could help us figure out one day if this limitation is lifted, for example.